### PR TITLE
LSAY analysis

### PR DESCRIPTION
**Pull request by tsat0002**
initial analysis of LSAY data - 2003

### CODE REVIEW

**Comment by nickjparkinson** [comment]
Hi Tash! I've had a quick look and don't have any substantial feedback yet. I'll dive in a little deeper later this week. Great work in your first six days. Super pleased with how much progress you've made.

**Comment by nickjparkinson** [comment]
Keep working on your branch for now and I'll review the PR and merge once ready.

**Comment by tsat0002** [comment]
I was able to figure out what was going on with the data frame - the strings in `Benchmark` were formatted slightly different, it works now.
 [c4b8585](https://github.com/grattan/school-ed-2025-primary-maths/pull/1/commits/c4b8585049fcaeef5b701499814d61c663431a66) has the tables for the averages across the years and honestly I think `intsvy` does a bit better of a job than the `survey` package (clear reduction across the states over 03, 06, 09).

File: LSAY_EDA.qmd [hunk 1]
```diff
@@ -0,0 +1,961 @@
...
+
+overall_mean_math_pv #using survey package method
+overall_school_mean #using intsvy package method
+
+
+```
+
+There is a discrepancy between the two different methods.
+
+Seems as though the level 3 proficiency levels for mathematics literacy is 482 in later years as well. \* [2018](https://nces.ed.gov/surveys/pisa/pisa2018/pdf/MathProfLevelDescriptionV2.pdf) \* [2022](https://nces.ed.gov/surveys/pisa/pisa2022/docs/DescriptionsOf2022ProficiencyLevels-TableI31.pdf)
```
Comment (nickjparkinson) [inline LSAY_EDA.qmd:334]: See my changes upstream, where I estimate the mean and standard deviation using the Rrepest package. This method is simpler and the most accurate method. The result is very close to your estimates using intsvy. The difference is explainable by the fact that I've weighted each replicate, whereas you've used a final weight.

File: LSAY_EDA.qmd [hunk 2]
```diff
@@ -0,0 +1,961 @@
...
+
+```{r}
+library(survey)
```
Comment (nickjparkinson) [inline LSAY_EDA.qmd:337]: See my code for how I've figured out the percentage who are not proficient. Note the large standard error on my estimates. This is important when working with a small sample. I get a very similar result (13 per cent), but it's using a slightly more robust method.

**Review by nickjparkinson — CHANGES_REQUESTED** [review]
Hi Tash!

This code is brilliant. I'm so impressed with how far you've got. As a next step, please see my changed and incorporate my upstream pull request. 

I'd then recommend going back over each cohort, and adapting my code which uses the Rrepest package.

**Pull request merged**